Dockerfile

```diff
@@ -30,7 +30,7 @@ RUN pip install -r requirements.txt
 COPY . /app
 
 RUN pip install -e . && \
-    pip install pyinstaller==3.5 && \
+    pip install pyinstaller==4.2 && \
```

Comment:
The app builds are still pinned to 3.5 version of PyInstaller and we are using the same versions across the application. We have not tested if everything works fine with4.2 so it is better to revert to 3.5 until we can test and make sure that everything works correctly.